README.md

```diff
@@ -642,7 +643,9 @@ field will be empty. If no operations has been called for a
 space, it will not be represented. Operations called for
 spaces not found both on storages and in statistics registry
 (including cases when crud can't verify space existence)
-will increment `space_not_found` counter.
+will increment `space_not_found` counter. Operation calls
+that reloaded a schema will increment `schema_reloads`
+counter.
```

Comment:
Strictly speaking this metrics is not about schema reloads, but rather about amount of retries of crud requests performed due to a stale schema on router. I'll illustrate it on the following case.

Say, we have space A and space B. Space A is changed somehow. We issue a request against space B. The router issues a request to a storage, the storage executes it, net.box stores result of the call, reloads the schema (on its own), returns the result of the call to router's crud code and the crud request is successfully executed. No explicit schema reload, no increment of the metrica.

However there is a latency spike due to the implicit schema reload (at least without `is_async = true`, but likely with it as well). I doubt it worth to really set `on_schema_reload` triggers and try to make the metric actually track schema reloads. I would describe it more precisely instead to don't confuse a user. Maybe even rename.

It also worth to note that sharding keys reload is not tracked here.

Further reading:

* https://github.com/tarantool/crud/pull/111
* https://github.com/tarantool/tarantool/issues/6169
* https://github.com/tarantool/crud/issues/253